apps/condo/domains/billing/constants.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Generated by `createservice billing.ValidateQRCodeService --type mutations`
+ */
+
+const BILLING_VALIDATE_QR_CODE_WINDOW = 24 * 60 * 60 // seconds
+const MAX_CLIENT_VALIDATE_QR_CODE_BY_WINDOW = 100
```

Comment:
I can't get the meaning of this constant

Reply:
The client may use this mutation not more than 100 times in 24 hours. These values may be tuned in the future.